README.md

```diff
@@ -1,29 +1,97 @@
-# Package Support Framework
-This project provides tools, libraries, documentation and samples for creating app-compat fixups to enable classic Win32 applications to be distributed and executed as packaged apps.
+# PSF Launcher

```

Comment:
It looks like the readme for the PSFMonitor is here.  If you want to add please append to the file don't overwrite it.

Reply:
restored original